lambda_agent.go

```diff
@@ -163,8 +163,9 @@ func (a *lambdaAgent) sendRequest(req *http.Request) error {
 			return nil
 		}
 
-		a.logger.Info("serverless agent has responded with ", resp.Status, ": ", string(respBody))
-		return nil
+		err = fmt.Errorf("serverless agent has responded with %s : %s", resp.Status, string(respBody))
```

Comment:
Suggestion: Better to move to another PR as it has functional implications

Reply:
done 👍